examples/fnn/mnist_conv_fnn.ml

```diff
@@ -5,15 +5,21 @@ open Tensorflow_fnn
 
 let label_count = Mnist_helper.label_count
 let image_dim = Mnist_helper.image_dim
-let epochs = 2000
 let batch_size = 256
+let steps = 2000
 
 let conv2d =
   Fnn.conv2d ~w_init:(`normal 0.1) ~filter:(5, 5) ~strides:(1, 1) ~padding:`same
 let max_pool = Fnn.max_pool ~filter:(2, 2) ~strides:(2, 2) ~padding:`same
 
 let () =
   let mnist = Mnist_helper.read_files () in
+  let epochs =
```

Comment:
Could you just hardcode `epoch` to the proper value here rather than deriving it from `steps` ? That would make things simpler.
Also could you adapt the other examples in the same directory ?

Reply:
I will do that. However the amount of epochs does indeed depend on the training set. But I can hard code it assuming training set is 60.000 samples.

Reply:
Sure hardcoding to the actual number of samples in the dataset is what I meant.